package.json

```diff
@@ -17,7 +17,7 @@
     "chai": "^3.4.1",
     "express": "^4.16.3",
     "mocha": "^5.2.0",
-    "puppeteer": "^1.8.0",
+    "puppeteer": "^1.16.0",
```

Comment:
Had to do a `npm audit fix ` , due to 2 high severity vulnerabilities (found by npm audit)